curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d3952f6af37b6a1c241c2.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-60
 
 # --description--
 
-Now that your `.map()` function is receiving the innermost function reference from `addCharacters`, it will properly iterate over the elements and pass each element as `n` to that function.
+Now that your `.map()` function is receiving the returned `num =>  charRange(...).map(...)` function reference from the curried `addCharacters` calls, it will properly iterate over the elements and pass each element as `n` to that function.
```

Comment:
```suggestion
Now that your `.map()` function is receiving the returned `num => charRange(...).map(...)` function reference from the curried `addCharacters` calls, it will properly iterate over the elements and pass each element as `n` to that function.
```